src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -25,57 +28,61 @@ public class ToBase64Transform : ICryptoTransform
 
         public int TransformBlock(byte[] inputBuffer, int inputOffset, int inputCount, byte[] outputBuffer, int outputOffset)
         {
-            ValidateTransformBlock(inputBuffer, inputOffset, inputCount);
+            // inputCount != InputBlockSize is not allowed
+            ThrowHelper.ValidateTransformBlock(inputBuffer, inputOffset, inputCount, InputBlockSize);
+            if (outputBuffer == null) ThrowHelper.ThrowArgumentNull(ThrowHelper.ExceptionArgument.outputBuffer);
 
             // For now, only convert 3 bytes to 4
-            byte[] tempBytes = ConvertToBase64(inputBuffer, inputOffset, 3);
+            Span<byte> input = inputBuffer.AsSpan(inputOffset, InputBlockSize);
+            Span<byte> output = outputBuffer.AsSpan(outputOffset, OutputBlockSize);
+
+            OperationStatus status = Base64.EncodeToUtf8(input, output, out int consumed, out int written, isFinalBlock: false);
+
+            if (written != OutputBlockSize)
+                ThrowHelper.ThrowCryptographicException();
```

Comment:
In crypto code braces are required for all blocks after an argument validation preamble.

Reply:
Since there are more stringent formatting rules for crypto it might be worth having a nested editor config file for those assemblies.